components/Header.tsx

```diff
@@ -11,24 +11,26 @@ import Search, { SearchProps } from './Search';
 const SearchBar = (props: SearchProps) => {
   const router = useRouter();
 
-  return (router.pathname === '/repos/[language]' && (
+  return (router.pathname === '/repos/[language]' ?(
       <Search {...props} />
     )
+    : 
+     <div className="hidden sm:inline-flex flex-1 max-w-md px-6 md:ml-5"></div>
```

Comment:
Can you use something else here because of this extra whitespace?

![image](https://github.com/max-programming/hacktoberfest-projects/assets/51731966/3471f97e-9348-4ddc-89f0-43e04946ef96)

Reply:
Removed extra white space